sys-apps/razercfg/razercfg-0.42.ebuild

```diff
@@ -77,8 +77,8 @@ src_install() {
 
 pkg_postinst() {
 	if use udev ; then
-		udevadm control --reload-rules
-		udevadm trigger --subsystem-match=usb
+		udevadm control --reload-rules || die
```

Comment:
Not sure if dying in middle of postinst is a good idea.

Reply:
I thought every call must "die" somehow - if not needed will remove.

Reply:
postinst is special. It's run *after* the package is installed, so it can't prevent it from being installed. Here's better to either ignore errors if they don't affect further code, or use things like `&&` to connect things that are interdependent. In either case, it's not for you to worry right now ;-).